### DISCUSSION TITLE
Add support for updating JsDelivr CDN links in HTML

### DISCUSSION THREAD

**Discussion by soenneker** [renovatebot/renovate#26935] [Suggest an Idea]
### Tell us more.

I believe Renovate automatically detects and updates cdnjs links... it would be nice if it supported jsdelivr. The libraries on jsdelivr tends to update much more quickly than cdnjs.

**Comment by rarkins** [comment]
I created https://github.com/renovatebot/renovate/issues/26936 for the datasource part and https://github.com/renovatebot/renovate/issues/26937 for the html manager part
